Resources/Prototypes/Guidebook/ss14.yml

```diff
@@ -3,6 +3,7 @@
   name: guide-entry-ss14
   text: "/ServerInfo/Guidebook/SpaceStation14.xml"
   children:
+  - Rules
```

Comment:
i'd say remove this since we already have a tab that lets you read the ingame rules